ext/cairo/rb_cairo_font_face.c

```diff
@@ -121,6 +121,46 @@ cr_font_face_quartz_supported_p (VALUE klass)
 #endif
 }
 
+#if CAIRO_HAS_FT_FONT
+static void
+handle_ft_error(FT_Error error)
+{
+#undef __FTERRORS_H__
```

Comment:
Can you clarify what this is needed for? I got the usage from the recommendation for getting the error message strings in the [source](https://github.com/vinniefalco/FreeType/blob/26568611186ca19e8b18afd2a9b0c2ce6c188165/include/freetype/fterrors.h#L64-L79).